@coderabbitai , what's wrong with WORMFC building? 

Starts here, check building logs, propose simple concise fix in unified diff  format.  Check if it affects other flight controllers as well, I am away from my AI agent. 

./src/main/drivers/barometer/barometer_lps.c:193:12: error: void value not ignored as it ought to be

--- a/src/main/drivers/barometer/barometer_lps.c
+++ b/src/main/drivers/barometer/barometer_lps.c
@@ -190,11 +190,13 @@ static uint32_t rawP = 0;
 static uint16_t rawT = 0;
 
 bool lpsWriteCommand(extDevice_t *dev, uint8_t cmd, uint8_t byte) {
-    return spiWriteReg(dev, cmd, byte);
+    spiWriteReg(dev, cmd, byte);
+    return true;
 }
 
 bool lpsReadCommand(extDevice_t *dev, uint8_t cmd, uint8_t *data, uint8_t len) {
-    return spiReadRegBuf(dev, cmd | 0x80 | 0x40, data, len);
+    spiReadRegBuf(dev, cmd | 0x80 | 0x40, data, len);
+    return true;
 }
 
 bool lpsWriteVerify(extDevice_t *dev, uint8_t cmd, uint8_t byte) {

